scenario_specifcation

```diff
@@ -0,0 +1,96 @@
+eg01-23 (Pu)
```

Comment:
Yikes. Another poor answer: the upcoming PR makes this into a markup file along with many other documentation from the Evaluation and Screening Report. [here](https://github.com/jbae11/transition-scenarios/blob/scenario_specification/scenario_specifcation.md)